### DISCUSSION TITLE
paddleocr的det、rec模型路径在哪？

### DISCUSSION THREAD

**Discussion by liuzh1516** [PaddlePaddle/PaddleOCR#13213] [Q&A]
我初始化的时候指定了模型路径：
`ocr = PaddleOCR(det_model_dir='~/.paddleocr/whl/det/ch/ch_PP-OCRv4_det_server_infer',
                rec_model_dir='~/.paddleocr/whl/rec/ch/ch_PP-OCRv4_rec_server_infer')  `
但实际上我已经把~/.paddleocr/whl下的包都rm -rf掉了，重启系统，为什么仍然能正常识别图片文字

**Comment by liuzh1516** [comment]
有人帮解答一下吗，谢谢。我又把路径换成了项目路径下尝试，并且已经把server版的高精度det和rec模型下载到./models/ch中，但初始化pandleOCR时都会自动下载一个特定版本，但查看下载的位置又没有下载的东西，启动之后感觉根本没加载我指定的模型，还是默认的：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/6716576/b8eef4fc-7f16-45ae-bdc0-ed3991cfa76c)

**Reply by SWHL** [reply]
去看一下源码，debugy一下吧，写得比较清楚。
https://github.com/PaddlePaddle/PaddleOCR/blob/433677182f108c1be413ee8a92815bc13b205737/paddleocr.py#L630

**Reply by liuzh1516** [reply]
> 去看一下源码，debugy一下吧，写得比较清楚。
> 
> https://github.com/PaddlePaddle/PaddleOCR/blob/433677182f108c1be413ee8a92815bc13b205737/paddleocr.py#L630

好的，那我自己研究下，谢谢
